components/select/nz-select-top-control.component.ts

```diff
@@ -36,7 +36,10 @@ import { NzSelectService } from './nz-select.service';
   animations: [zoomMotion],
   changeDetection: ChangeDetectionStrategy.OnPush,
   encapsulation: ViewEncapsulation.None,
-  templateUrl: './nz-select-top-control.component.html'
+  templateUrl: './nz-select-top-control.component.html',
+  host: {
+    '[style.cursor]': 'nzShowSearch ? "inherit" : "pointer"'
```

Comment:
any reason add this line?

Reply:
Hmm.. I can't recall it's been too long... let me try.

Reply:
I couldn't replicate the issue I ran into... it might be fixed by other updates, so I just removed it.